csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/schema/Schema.scala

```diff
@@ -25,6 +25,8 @@ case class Quoted() extends GlobalDirective("quoted")
 
 case class TotalColumns(numberOfColumns: BigInt) extends GlobalDirective("totalColumns")
 
+case class IntegrityCheck(filepathColumn: String, includeFolder: Boolean) extends GlobalDirective("integrityCheck")
```

Comment:
I don't understand why you want a Global Directive, why are implementation specific command line options not enough?